### ISSUE TITLE
Loss Functions

### ISSUE THREAD

**Issue by nagaswethar** [XHUJOY/CycleGAN-tensorflow#61]
self.g_loss_a2b = self.criterionGAN(self.DB_fake, tf.ones_like(self.DB_fake))\ 
+ self.L1_lambda * abs_criterion(self.real_A, self.fake_A_) \
+ self.L1_lambda * abs_criterion(self.real_B, self.fake_B_) \
+ self.Lg_lambda * gradloss_criterion(self.real_A, self.fake_B, self.weighted_seg_A) \
+ self.Lg_lambda * gradloss_criterion(self.real_B, self.fake_A, self.weighted_seg_B)\
self.g_loss_b2a = self.criterionGAN(self.DA_fake, tf.ones_like(self.DA_fake))\ 
+ self.L1_lambda * abs_criterion(self.real_A, self.fake_A_)\ 
+ self.L1_lambda * abs_criterion(self.real_B, self.fake_B_) \
+ self.Lg_lambda * gradloss_criterion(self.real_A, self.fake_B, self.weighted_seg_A)\ 
+ self.Lg_lambda * gradloss_criterion(self.real_B, self.fake_A, self.weighted_seg_B)\

can you explain this loss function and what \ indicates in code
